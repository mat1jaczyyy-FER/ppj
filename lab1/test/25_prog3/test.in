za i od 3 do 15
  a = i + i
  za j od a do 2*a
    b = j *j
    aza = 3
  az
az
