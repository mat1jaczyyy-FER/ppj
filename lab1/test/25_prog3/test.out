KR_ZA 1 za
IDN 1 i
KR_OD 1 od
BROJ 1 3
KR_DO 1 do
BROJ 1 15
IDN 2 a
OP_PRIDRUZI 2 =
IDN 2 i
OP_PLUS 2 +
IDN 2 i
KR_ZA 3 za
IDN 3 j
KR_OD 3 od
IDN 3 a
KR_DO 3 do
BROJ 3 2
OP_PUTA 3 *
IDN 3 a
IDN 4 b
OP_PRIDRUZI 4 =
IDN 4 j
OP_PUTA 4 *
IDN 4 j
IDN 5 aza
OP_PRIDRUZI 5 =
BROJ 5 3
KR_AZ 6 az
KR_AZ 7 az
